pages/common/cmake.md

```diff
@@ -1,13 +1,21 @@
 # cmake
 
-> Cross-platform build system generator.
-> It generates Makefiles, Visual Studio projects or others, depending on the target system.
+> Cross-platform build, testing and packaging automation system.
+> CMake uses its own shell-like syntax and generates recipes for many build systems.
```

Comment:
I'm afraid this second line like makes things too generic to be useful for the uninitiated. I like how the previous version was more concrete and offered specific examples that allow more people to understand what this is about.

I'm open to being convinced otherwise, though :)

Reply:
I'd prefer the prev version too. @bl-ue, @waldyrious, my friends, it is for your decision.

Reply:
I say go back to the old version then. 👍🏻

Reply:
Oh, sorry, I was dissipated, I meant other version. Initial one seems inaccurate for me. CMake does not generate build systems and projects. What about the following statement?
```Markdown
Cross-platform build automation system, that generates recipes for native build systems.
```

Reply:
Sounds good.  Go for it.